recipes/pyscss/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyScss" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "14a25c33c221a66bb1f000a6a067f376528d3df2f9333cee9c95709a9280cdb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
+  entry_points:
+    - pyscss=scss.tool:main
+    - less2scss=scss.less2scss:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - enum34
+    - pathlib
```

Comment:
Please drop anything from `build` that is not strictly needed for building. Probably should just be `python` and `setuptools`.